app/services/audit/checksum_validator.rb

```diff
@@ -1,296 +1,82 @@
 # frozen_string_literal: true
 
 module Audit
-  # Service for validating Moab checksums on storage, updating the MoabRecord db record, and reporting results.
+  # Service for validating Moab checksums in a directory. This class's primary collaborator
+  # is Audit::ChecksumValidationOrchestrator, which depends on Audit::ChecksumValidator to
+  # validate manifest inventories on disk and update the passed-in `results` object. It is
+  # also invoked by the `prescat:audit:validate_uncataloged` rake task which expects the
+  # class to emit validation results and errors to STDOUT.
   class ChecksumValidator
```

Comment:
This class no longer depends on the app DB. It *only* handles validating a Moab on disk.